kemeny_sourcemash.tex

```diff
@@ -43,15 +43,55 @@ \section{Project Overview}
 
 \section{User Instructions}
 
-While 
-
-\subsection{Production: www.sourcemash.com}
-
-When
-
-\subsection{Development: localhost}
-
-When
+Sourcemash is a web application that is hosted online at \url{www.sourcemash.com}. The current development release is available at \url{staging.sourcemash.com}, and the source code for Sourcemash is available at \url{https://github.com/sourcemash/Sourcemash}. Instructions for installation, execution, and testing in the local environment are included in the README and in �2.2 below.
+
+\subsection{Production}
+
+Users are encouraged to use the production release of Sourcemash available at \url{www.sourcemash.com}. The website uses Material Design techniques to create an easy-to-navigate, responsive user experience. Users are encouraged to sign up or log in to take advantages of all of the features that Sourcemash has to offer, and a continuously available help menu is available by pressing the ``?'' button in the bottom right corner of every page.
```

Comment:
The ? button and sign-in options are clear from visiting the site. I don't think we need to include them in the feature overview. I like how you used the navigation menu to walkthrough the site's core features.